templates/akeneo/files/.platform.template.yaml

```diff
@@ -21,9 +21,11 @@ info:
   # Unique machine name, prefaced by a vendor or organization identifier
   id: platformsh/akeneo
   # The human-readable name of the template.
-  name: Akeneo PIM
+  name: Akeneo PIM Community Standard Edition
```

Comment:
Working with Otavio we found that some titles are too long for the UI. Maybe Akeneo PIM CSE?

Reply:
CSE does not mean anything to me, maybe just Akeneo PIM and move the "Community Standard Edition" to the description.

Reply:
Shortened to "Community Edition", as that's what we call Magento.